deploy/helm/kubecf/values.yaml

```diff
@@ -175,6 +175,11 @@ sizing:
   diego_api:
     instances: ~
   diego_cell:
+    ephemeral_disk:
+      # Size of the ephemeral disk used to store applications
```

Comment:
Comment could say that the value is supposed to be in MB, but maybe this is obvious from the default value.